pkg/conversation/scrub_streaming_test.go

```diff
@@ -0,0 +1,1153 @@
+/*
```

Comment:
Do not use underscores in file names (except with `_test.go`).

Reply:
and mock based on the rest of the repo :)